docs/features/metrics.md

```diff
@@ -85,7 +82,7 @@ distinct-registrations: 30104 @ 2020-04-01 05:12:18
 null-registrations:
   sql: |
     select
-      count(distinct user_id)
+      count(*)
     from mart.user_signups
     where user_id is null
```

Comment:
I thought the number of null registrations should be `count(*)` instead of `count(distinct user_id)`, correct? `select count(distinct user_id) from ... where user_id is null` should always be either 0 or 1, isn't it?

Reply:
Haha good point. Didn't pay too much attention to this level of detail.